--- rt2x00-cvs-2006062902/radiobtn/radiobtn.c~	2006-06-01 00:17:48.000000000 +0200
+++ rt2x00-cvs-2006062902/radiobtn/radiobtn.c	2006-06-29 11:49:42.414903500 +0200
@@ -101,13 +101,16 @@
 	radiobtn->input_dev->id.bustype = BUS_HOST;
 	set_bit(KEY_RADIO, radiobtn->input_dev->keybit);
 
+
+	input_register_device(radiobtn->input_dev);
+/***	returns void in kernels < 2.6.15
 	status = input_register_device(radiobtn->input_dev);
 	if (status) {
 		printk(KERN_ERR "Failed to register input device %s.\n",
 			radiobtn->dev_name);
 		input_free_device(radiobtn->input_dev);
 		return status;
-	}
+	} */
 
 	/*
 	 * Set the initial state of the button.
