--- rt2x00-cvs-2007031214/config.orig	2007-03-02 19:31:20.000000000 +0100
+++ rt2x00-cvs-2007031214/config	2007-03-12 20:39:02.461201356 +0100
@@ -22,12 +22,12 @@
 # Enable rt2400pci support
 CONFIG_RT2400PCI=y
 # Enable rt2400pci hardware button support (requires rfkill)
-CONFIG_RT2400PCI_BUTTON=n
+CONFIG_RT2400PCI_BUTTON=y
 
 # Enable rt2500pci support
 CONFIG_RT2500PCI=y
 # Enable rt2500pci hardware button support (requires rfkill)
-CONFIG_RT2500PCI_BUTTON=n
+CONFIG_RT2500PCI_BUTTON=y
 
 # Enable rt2500usb support
 CONFIG_RT2500USB=y
@@ -35,7 +35,7 @@
 # Enable rt61pci support
 CONFIG_RT61PCI=y
 # Enable rt61pci hardware button support (requires rfkill)
-CONFIG_RT61PCI_BUTTON=n
+CONFIG_RT61PCI_BUTTON=y
 
 # Enable rt73usb support
 CONFIG_RT73USB=y
@@ -74,6 +74,6 @@
 #
 
 # Enable rfkill support
-CONFIG_RFKILL=n
+CONFIG_RFKILL=y
 # Enable rfkill asm file creation
 CONFIG_RFKILL_ASM=n
